examples/reference-entity-model/entities/Order.entity.json

```diff
@@ -1,31 +1,46 @@
-{
-  "info": {
-    "title": "Order",
-    "version": "0.0.1",
-    "baseUri": "http://marklogic.com/example/"
-  },
-  "definitions": {
-    "Order": {
-      "required": [ ],
-      "primaryKey": "orderId",
-      "properties": {
-        "orderId": {
-          "datatype": "string"
-        },
-        "address": {
-          "$ref": "#/definitions/Address"
-        }
-      }
-    },
-    "Address": {
-      "properties": {
-        "city": {
-          "datatype": "string"
-        },
-        "state": {
-          "datatype": "string"
-        }
-      }
-    }
-  }
-}
+{
+  "info": {
+    "title": "Order",
+    "version": "0.0.1",
+    "baseUri": "http://marklogic.com/example/"
+  },
+  "definitions": {
+    "Order": {
+      "primaryKey": "orderId",
+      "properties": {
+        "orderId": {
+          "datatype": "integer"
+        },
+        "orderTime": {
```

Comment:
Let's call this orderDateTime; across all of DHF, when possible, I'd like to use "Date" / "Time" / "DateTime" as a suffix when the type is date/time/dateTime.